src/importexport/capella/internal/capella.cpp

```diff
@@ -843,11 +845,12 @@ static Fraction readCapVoice(Score* score, CapVoice* cvoice, int staffIdx, const
             ClefType nclef = o->clef();
             LOGD("%d:%d <Clef> %s line %d oct %d clef %d",
                  tick.ticks(), staffIdx, o->name(), int(o->line), int(o->oct), int(o->clef()));
-            if (nclef == ClefType::INVALID) {
+            Measure* m = score->getCreateMeasure(tick);
+            if (nclef == ClefType::INVALID || nclef == pclef || nclef == m->staff()->defaultClefType().concertClef) {
```

Comment:
I have serious doubts about whether `Measure::staff()` returns anything meaningful, since measures represent all staves, not one. But see https://github.com/musescore/MuseScore/pull/22670#discussion_r1590327760.

Reply:
Yep, it seems it just crash ;-)